### ISSUE TITLE
Be smarter about scala.js

### ISSUE THREAD

**Issue by nafg** [scalacenter/scala3-migrate#180]
I get a lot of these:

>
```
[info] org.scala-js:scalajs-compiler:1.5.0:plugin->default(compile) -> X : Scala 2 compiler plugins are not supported in scala 3.0.0-RC1. You need to find an alternative
[info] org.scala-js:scalajs-library:1.5.0                           -> "org.scala-js" % "scalajs-library_2.13" % "1.5.0"
[info] org.scala-js:scalajs-test-bridge:1.5.0:test                  -> "org.scala-js" % "scalajs-test-bridge_2.13" % "1.5.0" % "test"
```

However, these are added via the scalajs sbt plugin, and it's smart enough to not include the compiler plugin on 3.0.0-RC1 (since it's built in) and to add the right versions of its libraries.

So (at least when included via the sbt plugin) these warnings should be silenced. Or at least, they could be printed in a less "scary" way (like more compressed, and with a note saying "you can probably ignore this because ..."

**Comment by mlachkar** [comment]
You are right! There were not enough tests for scalajs projects, And I m planning to add a better support for Scala.js with more tests.
For the compiler plugin, we could suggest the new scalacOption `-scalajs` that replaces it.  Would it be ok ?

**Comment by nafg** [comment]
Again, I believe the sbt plugin handles that. The problem is that I don't know if your plugin can tell if the scalajs sbt settings were added by the plugin or by hand, or whether you care about the case where someone adds them by hand (since while that technically could be done, it probably is never done that way.

If you can tell, or are comfortable assuming, that the scalajs sbt plugin is present, then you don't have to emit any warning at all.

If you want to cover the case of manually added scalajs settings then you could suggest that. If don't know and want to print it "just in case" then make that clear.

To be clear, IIUC `-scalajs` replaces the compiler plugin. The other dependencies are needed (I'm not sure why it's picking 2.13 versions; is that how it's meant to work?) and so for the case of manually added sbt settings for scalajs those notices would be appropriate (assuming the 2.13 artifacts are the right ones to use). (Again, the sbt plugin would take care of that as well, of course.)

**Comment by mlachkar** [comment]
Hello, a new release has been published `0.4.4` that fixes this issue. If you have time to try and give some feedbacks, please do. Thanks again
